From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Sat, 30 Apr 2022 10:12:24 +0100
Subject: indent usage options

The --help text options should be indented so help2man parses them correctly.
---
 src/params.vala | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/src/params.vala b/src/params.vala
index 2e30887..24f2a9a 100644
--- a/src/params.vala
+++ b/src/params.vala
@@ -137,16 +137,16 @@ namespace Terminus {
 terminus [OPTION...] [-- COMMAND ...]
 
 Help commands:
--h, --help                    show this help
--v, --version                 show version
+  -h, --help                    show this help
+  -v, --version                 show version
 
 Options:
--x, --execute, --             launches a new Terminus window and execute the remainder of the command line inside the terminal
--e, --command=STRING          launches a new Terminus window and execute the argument inside the terminal
---working-directory=DIRNAME   sets the terminal directory to DIRNAME
---no-window                   launch Terminus but don't open a window
---nobindkey                   don't try to bind the Quake-mode key (useful for gnome shell)
---check-gnome                 exit if we are running it in Gnome Shell (guake mode should be managed by the extension)
+  -x, --execute, --             launches a new Terminus window and execute the remainder of the command line inside the terminal
+  -e, --command=STRING          launches a new Terminus window and execute the argument inside the terminal
+  --working-directory=DIRNAME   sets the terminal directory to DIRNAME
+  --no-window                   launch Terminus but don't open a window
+  --nobindkey                   don't try to bind the Quake-mode key (useful for gnome shell)
+  --check-gnome                 exit if we are running it in Gnome Shell (guake mode should be managed by the extension)
 """));
 	}
 
